apps/code-examples/src/app/code-examples/ag-grid/data-grid/data-manager/data-manager-data-grid-docs-demo-view-grid.component.ts

```diff
@@ -27,17 +29,13 @@ import {
   templateUrl: './data-manager-data-grid-docs-demo-view-grid.component.html',
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
-export class DataManagerDataGridsDocsDemoViewGridComponent implements OnInit {
+export class DataManagerDataGridDocsDemoViewGridComponent implements OnInit {
   @Input()
   public items: any[] = [];
 
   public viewId = 'dataGridWithDataManagerView';
 
-  public columnDefs: ColDef[] = [
-    {
-      field: 'selected',
-      type: SkyCellType.RowSelector,
-    },
+  public singleSelectColumnDefinitions: ColDef[] = [
```

Comment:
in these code examples I would keep these as `columnDefs` instead of single- or multiSelectColumnDefs since there is no toggling between the 2--consumers might be confused by the naming

Reply:
fixed in: [rename properties](https://github.com/blackbaud/skyux/pull/531/commits/807ce4513c477a7543e9ba9cca8700aa21ba2365)